src/auth-service/models/Maintenance.js

```
@@ -0,0 +1,234 @@
+          success: true,
+          message: "maintenance not created despite successful operation",
+          status: httpStatus.OK,
+          data: [],
+        };
+      }
+    } catch (err) {
+      handleError(err, next, "Internal Server Error");
+    }
+  },
+  async list(
+    { skip = 0, limit = Math.min(limit, maxLimit), filter = {} } = {},
```

Biome

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)